docs/CHANGELOG.asciidoc

```diff
@@ -64,6 +64,8 @@ APM::
 * Agent configuration GA {pull}46995[#46995]
 Canvas::
 * Adds ability to share workpads in other websites {pull}46278[#46278]
+Code::
```

Comment:
As this is a breaking change, can you please change it to the breaking changes section?